### ISSUE TITLE
[staging] Provide 'link copied' feedback in social sharing components

### ISSUE THREAD

**Issue by m-branson** [mozilla/voice-web#1153]
Currently there is no url copy confirmation feedback when clicking on the link icon in both the footer and social sharing overlay (during contribution flow). Let's provide feedback, when this has been clicked or tapped, that the link has been copied and is therefore ready for use.

Stylistically this could take the form of our black pill tooltip (removing the bottom arrow). This would show above the link icon once a successful url copy has been achieved.

Social overlay:
<img width="1054" alt="screen shot 2018-06-26 at 10 13 19 am" src="https://user-images.githubusercontent.com/18686986/41928321-aced9412-7929-11e8-8e2e-900bc543088f.png">

Footer:
<img width="678" alt="screen shot 2018-06-26 at 10 13 05 am" src="https://user-images.githubusercontent.com/18686986/41928330-b1db2cbe-7929-11e8-8487-5112862fe3cd.png">

Current tooltip styling:
<img width="318" alt="screen shot 2018-06-26 at 10 13 26 am" src="https://user-images.githubusercontent.com/18686986/41928339-b6fd1874-7929-11e8-9709-a82c9bd84415.png">

**Comment by Gregoor** [comment]
Added it to staging, will wait for feedback before getting it onto prod and closing this issue.

**Comment by m-branson** [comment]
Thanks @Gregoor, similar feedback to the clip upload success toast:

- [ ] Green pill background with white text
- [ ] Use the link icon (white on the green background) with white `Link Copied`text
- [ ] Possible to add a fade in when it appears? Currently it's quite abrupt
- [ ] The location of the toast in relation to the footer link icon is odd. The color may help with this but curious if it's possible to have the link toasts (footer & social) appear just above the icon click it is toasting? ...realizing that this mean it's no longer a 'global toast' UI. 😬

**Comment by Gregoor** [comment]
Thanks for the feedback! I made 3/4 changes. The positioning one I'm a bit unsure about. Maybe we should use tooltips for it instead then (also entirely possible, just a different UI component then imo). Having it more local also kind of makes sense to me

**Comment by m-branson** [comment]
Hey this is looking & working great; I've removed my 4/4 because I think we're good without it. Can keep an eye out and see how people fair.

The only bug I've found is in the social overlay; when I click the link icon, the toast is under the overlay background (barely visible in this screenshot). Possible to bring above so it's noticeable?

![screen shot 2018-07-10 at 10 53 29 am](https://user-images.githubusercontent.com/18686986/42528289-de4900ee-842f-11e8-8fab-635138d9df06.png)
